src/commands/add-formatting.js

```diff
@@ -0,0 +1,134 @@
+const lang = require('language-classifier');
+const hljs = require('highlight.js');
+const confusedMessageGenerator = require('../utilities/confusedMessageGenerator');
+const isSuppotedByPrettier = require('../utilities/isSuppotedByPrettier');
+const formatter = require('../utilities/formatter');
+
+module.exports = {
+  prefix: '',
+  /**
+   * @name add-formatting
+   * Inserts any unformated code to a code block, enables syntax highlighting and formats it when a
+   * message gets a specific reaction
+   *
+   * @param {Discord.Message} message the message provided
+   */
+  command: function addFormatting(message) {
+    const { content } = message;
+
+    const languageGuesses = [];
+
+    const detectedWithHLJS = hljs.highlightAuto(message.content);
+    const detectedWithLanguageClassifier = lang(message.content);
+
+    languageGuesses.push(detectedWithHLJS.language);
+    if (detectedWithHLJS.second_best) {
+      languageGuesses.push(detectedWithHLJS.second_best.language);
+    }
+    languageGuesses.push(detectedWithLanguageClassifier);
+
+    console.log(languageGuesses);
+
+    console.log(detectedWithLanguageClassifier, detectedWithHLJS);
+    if (!detectedWithHLJS.top.aliases) {
+      detectedWithHLJS.top.aliases = [];
+    }
+    if (
+      detectedWithLanguageClassifier === detectedWithHLJS.language ||
+      detectedWithHLJS.top.aliases.includes(detectedWithLanguageClassifier) // need compare it with hljs's second guess here too
+    ) {
+      let supportedLanguage = isSuppotedByPrettier(
+        detectedWithLanguageClassifier
+      );
+
+      if (supportedLanguage) {
+        const formattedCode = formatter(content, supportedLanguage);
+        message.channel.send(
+          `\`\`\`${detectedWithLanguageClassifier}\n${formattedCode}\`\`\``
+        );
+      } else {
+        message.channel.send(
+          `\`\`\`${detectedWithLanguageClassifier}\n${content}\`\`\``
+        );
+      }
+    } else {
+      const reactionEmojies = ['🙂', '🙃', '😃'];
```

Comment:
Instead of using an array here and then referencing each emoji via their index, we can use an enum-like approach (the keys of the "enum" can be named to whatever makes sense, or match with their actual name):
```javascript
const reactioEmojies = {
  good: '🙂',
  bad: '🙃',
  great: '😃'
};
//..
switch(reactionEmoji) {
  case 'good':
 //..
```

Reply:
Done..... fixed it!